docs/built_rst/xml/elements/election.rst

```diff
@@ -6,7 +6,7 @@ Election
 ========
 
 The Election object represents an Election Day, which usually consists of many individual contests
-and/or referenda. A feed may **not** contain multiple Election objects. All relationships in the
+and/or referenda. A feed must contain **exactly one** Election object. All relationships in the
```

Comment:
Interesting. Is this a suggestion from Sophie re: clarity in technical guidance?